types/Event/Event.d.ts

```diff
@@ -70,7 +70,7 @@ export declare class Event implements IEvent {
    * Retrieve all event objects.
    *
    * @param {Object} params - The parameters to use for the request.
-   * @returns {Promise<Event>} The list of {@link Event}.
+   * @returns {Promise<Event>} The list of {@link Event events}.
```

Comment:
These don't actually return an Event, or even a list of Events, but an Object that has two properties. Please update for all "all" functions accordingly.